src/ResponsiveProvider.js

```diff
@@ -4,47 +4,40 @@ import PropTypes from 'prop-types';
 import React, { useMemo, useRef, useState } from 'react';
 import ResponsiveContext from './ResponsiveContext';
 
-const ResponsiveProvider = ({
-    initialMediaType,
-    defaultOrientation,
-    children,
-    breakpoints,
-    breakpointsMax,
-    mediaQueries,
-}) => {
-    const [isCalculated, setIsCalculated] = useState(false);
-    const [currentBreakpoint, setCurrentBreakpoint] = useState(
-        initialMediaType
-    );
+const useOrientation = (defaultOrientation) => {
     const [currentOrientation, setCurrentOrientation] = useState(
         defaultOrientation
     );
 
-    const breakpointsWithInitialValue = {
-        _initial: '0em',
-        ...breakpoints,
-    };
-
-    const breakpointNames = Object.keys(
-        mediaQueries || breakpointsWithInitialValue
-    );
+    // Add orientation media query
```

Comment:
We already now by the hook that this is to deal with the orientation I think we don't need this comment anymore, what you think?

Reply:
Good point, changed, and deleted the comment from the other hook as well.  What do you think?

Reply:
Yap good to me